src/AzureSqlAction.ts

```diff
@@ -70,6 +94,30 @@ export default class AzureSqlAction {
         console.log(`Successfully executed Sql file on target database.`);
     }
 
+    private async _executeBuildProject(inputs: IBuildAndPublishInputs): Promise<string> {
+        const projectName = path.basename(inputs.projectFile, Constants.sqlprojExtension);
+        const additionalBuildArguments = inputs.buildArguments ?? '';
+        const parsedArgs = DotnetUtils.parseCommandArguments(additionalBuildArguments);
+        let outputDir = '';
+
+        // Set output dir if it is set in the build arguments
+        const outputArg = DotnetUtils.findArgument(parsedArgs, "--output", "-o");
+        if (!outputArg) {
```

Comment:
shouldn't this be `if (outputArg)` since it should get set if it isn't undefined?

Reply:
No we're actually setting outputDir to this value if it was passed in. Renamed the variable to be more clear.